### DISCUSSION TITLE
The world is becoming more Headless, what are the future plans for Prestashop?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [PrestaShop/PrestaShop#33341] [Q&A] [answered]
Hi gall, the world is becoming more & more Headless, what are the future plans for Prestashop? Is there a roadmap I can check?
Thanks.

**Comment by vitalijalbu** [comment]
@eternoendless any replies on this please?? thanks

**Comment by eternoendless** [comment, marked as answer]
Hi @vitalijalbu this all has been laid out in my [series of articles](https://build.prestashop-project.org/news/2019/prestashop-in-2019-and-beyond-introduction/) (more particularly in the last two articles). There are plans to build a front office API in the future but I wouldn't be able to say when exactly.

**Reply by j-vitali** [reply]
but storefront at least? :D
